hles/transformation/src/main/scala/org/broadinstitute/monster/dap/ScioUtils.scala

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.monster.dap
+
+import com.spotify.scio.values.SCollection
+
+import io.circe.syntax._
+import io.circe.{JsonObject, Encoder}
+
+import org.broadinstitute.monster.common.StorageIO
+
+object ScioUtils {
+
+  def jsonSerializeTransform[M: Encoder](messages: SCollection[M], description: String) = {
+    messages.transform(s"Convert '$description' messages to JSON")(
+      _.map((msg: M) => msg.asJson.printWith(StorageIO.circePrinter))
+    )
+  }
+
+  def toJsonAndTsv[M: Encoder](
```

Comment:
🤔 I'm wondering if this should consume a `numShards` argument that we can pass in to `saveAsTextFile` so we can end up with a single TSV rather than having to stitch together >=1 after the job.

Reply:
+1